.github/workflows/tests.jsonnet

```diff
@@ -494,6 +494,26 @@ local push_docker_nonroot_job = {
   },
 };
 
+local build_test_docker_performance_tests_job = build_test_docker_nonroot_job + {
+  with: super.with + {
+    'docker-flavor': |||
+      latest=auto
+      suffix=-performance-tests,onlatest=true
```

Comment:
For my knowledge: How does adding this suffix request the performance tests?

Reply:
The suffix is added to the container image tags, which can then be pulled from Dockerhub:
![image](https://github.com/semgrep/semgrep/assets/2374948/dc6f5e88-c220-464c-aac9-b43dee647270)